src/renderer/components/table/table-cell.tsx

```diff
@@ -20,9 +23,30 @@ export interface TableCellProps extends React.DOMAttributes<HTMLDivElement> {
   _sorting?: Partial<TableSortParams>; // <Table> sorting state, don't use this prop outside (!)
   _sort?(sortBy: TableSortBy): void; // <Table> sort function, don't use this prop outside (!)
   _nowrap?: boolean; // indicator, might come from parent <TableHead>, don't use this prop outside (!)
+  _onResize?(width: number): void; // <Table> resize callbalck, don't use this prop outside (!)
 }
 
-export class TableCell extends React.Component<TableCellProps> {
+type ResizeHandlerState = {
+  mousePosX?: number,
+};
+
+/**
+ * Ergonomic rate for throttling mouse events
+ * when resizing, in miliseconds.
+ * Represents optimal balance between user-facing latency
+ * and rendering performance 
+ */
+const ERGONOMIC_THROTTLE_RATE = 10;
+
+export class TableCell extends React.Component<TableCellProps, ResizeHandlerState> {
+  private resizeHandlerState?: ResizeHandlerState;
+  private cellContainer: React.RefObject<HTMLDivElement> = React.createRef();
+
+  constructor(props: TableCellProps) {
+    super(props);
+    this.state = {};
```

Comment:
Not sure, but maybe it's better to use `mobx` everywhere from this point for state management in react-components?

Reply:
I'll be sure to adopt it as a general practice in scope of the project, however right here it turned out to be more beneficial to keep `resizeHandlerState` outside component lifecycle all-together, since it is updated by global DOM-native events.